src/ai/backend/manager/registry.py

```diff
@@ -3045,7 +3053,11 @@ async def commit_session(
         Commit a main kernel's container of the given session.
         """
 
-        kernel = await self.get_session(session_name_or_id, access_key)
+        kernel = await self.get_session(session_name_or_id, access_key, allow_stale=True)
+        if kernel["status"] != KernelStatus.RUNNING:
+            raise InvalidAPIParameters(
+                f'Unable to commit since kernel(id: {kernel["id"]}) is currently not RUNNING.'
+            )
```

Comment:
Same here.
```suggestion
            raise InvalidAPIParameters(
                f"Unable to commit since kernel(id: {kernel["id"]}) is currently not RUNNING."
            )
```